service/static/js-src/muk.publisher.js

```diff
@@ -4618,44 +4620,43 @@ $.extend(true, edges, {
+                        val = ts.values[i]
+                        if (val.count !==0 || !this.hideEmpty) {
+                            if(($.inArray(val.term.toString(), ts.filters)) === -1 ) {
+                                results += '<div class="form-fields__item-checkbox ' + valClass
+                                results += '" data-key="' + edges.escapeHtml(val.term) + '"><label><input type="checkbox"/> ' + edges.escapeHtml(val.display)
+                                results += '<span class="' + countClass + '"> (' + val.count + ")</span>"
+                                results += "</label></div>"
+                            }
                         }
                     }
+                    results += '</div>';
                 }
+
                 var controlFrag = "";
                 if (this.controls) {
                     var ordering = '<a href="#" title=""><i class="glyphicon glyphicon-arrow-up"></i></a>';
-                    controlFrag = '<div class="' + controlClass + '" style="display:none" id="' + controlId + '"><div class="row">                         <div class="col-md-12">                            <div class="btn-group">                                <button type="button" class="btn btn-default btn-sm" id="' + sizeId + '" title="List Size" href="#">0</button>                                 <button type="button" class="btn btn-default btn-sm" id="' + orderId + '" title="List Order" href="#"></button>                             </div>                        </div>                    </div></div>'
+                    controlFrag = '<div class="' + controlClass + '" style="display:none" id="' + controlId + '"><div class="row"><div class="col-md-12"><div class="btn-group"><button type="button" class="btn btn-default btn-sm" id="' + sizeId + '" title="List Size" href="#">0</button><button type="button" class="btn btn-default btn-sm" id="' + orderId + '" title="List Order" href="#"></button></div></div></div></div>'
                 }
                 var filterFrag = "";
                 if (ts.filters.length > 0 && this.showSelected) {
                     for (var i = 0; i < ts.filters.length; i++) {
                         var filt = ts.filters[i];
-                        filterFrag += '<div class="' + resultClass + '"><strong>' + edges.escapeHtml(filt.display) + "&nbsp;";
-                        filterFrag += '<a href="#" class="' + filterRemoveClass + '" data-key="' + edges.escapeHtml(filt.term) + '">';
-                        filterFrag += '<i class="glyphicon glyphicon-black glyphicon-remove"></i></a>';
-                        filterFrag += "</strong></a></div>"
+                            results = '<div class="form ' + resultClass + '"><div class="form-fields__item-checkbox ' + filterRemoveClass + '" data-key="' + edges.escapeHtml(filt.term) + '"><label><input type="checkbox" checked/> ' + edges.escapeHtml(filt.display);
+                            results += "</label></div></div>"
```

Comment:
You have the problem here of multiple "form" divs, which is what the earlier code removed. E.g. see line 4626.